.gitlab-ci.yml

```diff
@@ -53,7 +55,7 @@ deploy_prod:
   script:
     - chmod u+x build/dist/${APP_NAME_VERSION}
     - scp -o StrictHostKeyChecking=no build/dist/${APP_NAME_VERSION} datalib@codon-login-01.ebi.ac.uk:${FTP_STG_DIR}
-    - ssh datalib@codon-login-01.ebi.ac.uk "bsub -q datamover cp ${FTP_STG_DIR}/${APP_NAME_VERSION} ${FTP_DIR}/ && cd ${FTP_DIR}/ && ln -sf ${APP_NAME_VERSION} ${APP_NAME}.zip"
+    - ssh datalib@codon-login-01.ebi.ac.uk "bsub -q datamover ./${SCRIPT_PATH}/${SCRIPT_NAME} ${FTP_STG_DIR} ${APP_NAME_VERSION} ${FTP_DIR} ${APP_NAME}.zip"
```

Comment:
why ./ ?
${FTP_STG_DIR} and ${FTP_DIR} and ${APP_NAME}.zip are fixed so can be in the script. only variable is the new artifact.

bsub should be in the script. not here.
u only call the script from here